not_mvp/1224-user_administration/1313-administrator_views_user_administration_sections_with_role_members_part_ii.feature

```diff
@@ -18,8 +18,8 @@ Feature: Administrator views User Administration Sections with Role members (Par
         And they see a checkbox for "Assign role Creator automatically"
         Examples:
             | Section                    | Role          |
-            | Institution Administrators | Administrator |
+            | Administrators             | Administrator |
             | Curators                   | Curator       |
             | Editors                    | Editor        |
-            | Creators                   | Creator       |
+            | Registrator                | Registrator   |
```

Comment:
Nå gikk det opp for meg at vi har en ufin miks med Creator/Registrator overalt :(

Reply:
ouch. tenkte ikke på det 😬

Reply:
Skal holde et øye ut for det når jeg går gjennom scenarioene